src/components/Content/RightPanel/TaskDetails/TaskTitle/TaskTitle.js

```diff
@@ -3,6 +3,12 @@ import React, { Component } from 'react';
 import './TaskTitle.css';
 
 class TaskTitle extends Component {
+  static autoResize() {
+    var el = document.getElementById('discuss-task-title-update');
+    el.style.height = "";
+    el.style.height = (el.scrollHeight + 2) + "px";
+  }
+
```

Comment:
@colinfruit To clarify my comments earlier, seems like we should be able to set the height using CSS instead here :+1: